.github/workflows/scripts/tag_latest_image.sh

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env bash
+
+export MY_ORG=$(echo "${GITHUB_REPOSITORY}" | awk -F / '{print $1}')
+
+if [ "${GITHUB_BASE_REF}" != "master" ]; then
+    export SUFFIX="-""${GITHUB_BASE_REF}"
+else
+    export SUFFIX=""
+fi
```

Comment:
Same logic for this tagging script -- if the branch is not master, append the branch name.